requirements.txt

```diff
@@ -4,3 +4,4 @@ south
 whoosh
 ipy
 python-ldap
+djangorestframework==2.4.1
```

Comment:
Don't pin the version, to allow some more leeway to people. Indeed let's avoid djangorestframework 3.X for now so

djangorestframework>=2.0.0,<3.0.0